.github/workflows/sub-build-docker-image.yml

```diff
@@ -75,6 +75,8 @@ jobs:
     permissions:
       contents: 'read'
       id-token: 'write'
+    env:
+      DOCKER_BUILD_SUMMARY: ${{ vars.DOCKER_BUILD_SUMMARY }}
```

Comment:
In your PR comment you said this is not supported by docker build cloud. Why are we adding it here?

Reply:
Oh, I'm adding it as a variable, which defaults to false, just in case is supported later on and we'd like to activate it.

Reply:
I've update the PR comment with this info.